crawler/reuters.py

```diff
@@ -78,10 +81,10 @@ def get_news_num_whenever(self, url):
         # return the number of news
         soup = util.get_soup_with_repeat(url, repeat_times=4)
         if soup:
-            return len(soup.find_all("div", {'class': ['topStory', 'feature']}))
+            return len(soup.find_all("div", {'class': ['item']}))
         return 0
 
-    def fetch_within_date_range(self, news_num, url, date_range, task, ticker):
+    def fetch_within_date_range(self, news_num: int, url: str, date_range: list, task: tuple, ticker: str):
```

Comment:
Date ranges are redundant now - Reuters lists all articles, and no longer has a date query parameter in the URL.

The code uses the dates to keep track of whether it needs to load new articles. This needs it be refactored out, and we should rely on the date as parsed from the news article itself. An ID is also available in the parsed article URL, so perhaps that could be useful for ensuring all news articles are unique.